src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,172 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
```

Comment:
Make `Node<T>` an inner class. Currently, `Node<T>` is a static nested class, but it should be moved inside the `MyLinkedList` class and should not be static.